CHANGELOG.rst

```diff
@@ -8,6 +8,135 @@
 
 .. towncrier release notes start
 
+Pytest 3.4.0 (2018-01-29)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- All pytest classes now subclass ``object`` for better Python 3 compatibility.
```

Comment:
s/2/3 ?!

Reply:
I meant "compatibility with Python 3", but I think changing to "Python 2/3 compatibility will be better" 👍